public/javascripts/SVLabel/src/SVLabel/navigation/MapService.js

```diff
@@ -645,7 +645,16 @@ function MapService (canvas, neighborhoodModel, uiMap, params) {
         var message = "Uh-oh, something went wrong with Google Street View. " +
             "This is not your fault, but we will need to move you to another location in the " +
             currentNeighborhoodName + " neighborhood. Keep up the good work!";
+        svl.panorama.set('linksControl', false);//disable arrows
+        disableWalking(); //disable walking
+        disablePanning(); //disable panning
+        svl.canvas.disableLabeling(); //disable labeling
+
         var callback = function () {
+            enableWalking(); //enable walking
```

Comment:
Enabling after the jump statement caused issues I believe (such as no being able to walk after clicking the pop-up).